src/DataHandling/Logs/DataDogAudit.php

```diff
@@ -87,13 +87,28 @@ protected function getAssociations($entity)
     /**
      * Method creating version key (to group table entries associated with the same user action).
      * @param \DataDog\AuditBundle\Entity\AuditLog $currentVersion entity for which version key is created
+     * @param array $diffArray subsequent elements are diff for each version
```

Comment:
stile hint: empty comment line (`\n*\n`) before `@param` and `@return`.
Not for fixing, but for doing next time.